tests-integration/reference-model/src/Morphir/Reference/Model/TypeAscription.elm

```diff
@@ -0,0 +1,16 @@
+module Morphir.Reference.Model.TypeAscription exposing (..)
+
+
+type Custom
+    = Ctor1
+    | Ctor2
+
+
+use : Bool
+use =
+    target Ctor1 Ctor2
```

Comment:
You should have multiple us examples passing constructors with various numbers of arguments. You should also have ones where there are unspecified arguments (meaning that the constructor has say 2 args but only 1 is passed in because the expected type is a function).

Reply:
For example:
```elm
use2 = 
    target (Ctor2 41)

use3 =
    target2 (Ctor3 1)

target2 : (a -> Int) -> Bool
target2 = 
    True
```

Reply:
done